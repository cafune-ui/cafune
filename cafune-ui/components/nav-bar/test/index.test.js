
  import { NavBar } from '../';
  import { render } from 'enzyme';
  import { shallow, deep } from 'preact-render-spy';
    
  describe('NavBar', () => {
    it('should render properly', () => {
      const wrapper = shallow(<NavBar />);
      expect(wrapper).toMatchSnapshot();
    });
  });