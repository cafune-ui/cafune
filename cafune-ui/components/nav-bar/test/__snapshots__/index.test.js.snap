// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render properly 1`] = `
RenderContext {
  "0": VNode {
    "attributes": undefined,
    "children": Array [],
    "key": undefined,
    "nodeName": undefined,
  },
  "length": 1,
}
`;
